_component_design/buttons.md

```diff
@@ -368,3 +368,57 @@ Split drop-down buttons are preferred when more than one action is associated wi
 </div>
 
 </section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow" markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Download / export buttons
+
+- Download links **are always** formatted as: Download [noun] as [format]. 
+- Download is a word that focuses about the user's benefit because there is no ambiguity about the destination (the user's platform) and it will then be theirs.
```

Comment:
s/`Download is a word that focuses about the user's benefit because there is no ambiguity about the destination (the user's platform) and it will then be theirs.`/`Use the word **Download** because there is no ambiguity about the destination (the user's platform). The word focuses on the user's benefit because the downloaded object becomes the user's.`

Reply:
updated.